### ISSUE TITLE
Right here.

### ISSUE THREAD

**Issue by El-Qassam** [El-Qassam/FreeFox#1]
Ashlyn, feel free to talk here like the old issue chat. A friend of mine was just spying on our conversation, so I had to throw it out. Be careful with whatever you type here cuz pretty much anyone with a clue can see it.

**Comment by El-Qassam** [comment]
Get on. And don’t annoy Mr. Waltz. He has some evidence.

**Comment by AJSagan** [comment]
Okay. So first you tell me this website is good. Then you say it isn’t likely anybody will see anything? Then next thing you know, some rando has our entire chat history? And even further, you then ask me to keep using it? 

Dude. Just use the document.

**Comment by El-Qassam** [comment]
> Okay. So first you tell me this website is good. Then you say it isn’t likely anybody will see anything? Then next thing you know, some rando has our entire chat history? And even further, you then ask me to keep using it? As you would say, I’ll just “have no recollection of this event”.
> 
> Dude. Just use the document.

1. If you don’t wanna use this, fine, but I’m not writing stuff on a Google Doc. We don’t really have much to say during school anyways, and we have WhatsApp.
2. He’s not a rando and stop typing stuff that he may or may not be seeing this very instant.
3. Perfect. If anyone approaches you about anything, just play dumb for as long as possible (infinite). What do you think I’ve been doing?

**Comment by AJSagan** [comment]
Fine, Mr. My Friends are Nice People But They Still Spy on Me. If they were truly “your bois” they would just let you be.

**Comment by AJSagan** [comment]
Fine. I’m leaving for practice. Let’s talk there.

**Comment by El-Qassam** [comment]
> Fine. I’m leaving for practice. Let’s talk there.

Good good. Did I not just say not to mention sensitive info though? That involves activities. Just use this for school for short notices.

**Comment by AJSagan** [comment]
Sure. Do you have the Language Arts homework?

**Comment by El-Qassam** [comment]
> Sure. Do you have the Language Arts homework?

Yeah, I sent it.

**Comment by El-Qassam** [comment]
Sorry I couldn’t be at practice, I was with the bois in Acadmic Supa Bowl.

**Comment by AJSagan** [comment]
You’re good. I saw you leaving with a short little guy.

**Comment by El-Qassam** [comment]
> You’re good. I saw you leaving with a short little guy.

Navindu? Short Indian dude. That’s my son.

**Comment by AJSagan** [comment]
Your son from whom. I should know about this…
Also, I just finished my paragraph for the Spanish project.

**Comment by El-Qassam** [comment]
> Your son from whom. I should know about this…
> Also, I just finished my paragraph for the Spanish project. I texted you it.

My adopted son, what are ya talkin bout? Yeah I got it, Imma add it soon.

**Comment by AJSagan** [comment]
Interesting. You haven’t mentioned a son before. I don’t know if I’m fathering a son anytime soon. So could you come over during Tech? Tell me when you’re there so I go to the commons.

**Comment by El-Qassam** [comment]
I’ll leave in a sec.

**Comment by AJSagan** [comment]
> OK, so recapping on our conversation. First, read the texts I sent you. You know they don’t mean it. They couldn’t hurt a fly and they have the IQ of a gerbil. Nothing wrong in seeing them, and if we tell them not to tell anyone, I’m sure they would — with some reluctance. I can assure you.
> 
> As for what happened after practice, I don’t think that’s gonna happen. Not again. And if it does, well, we’d do the responsible thing, right? No harm done whatsoever. Text me your thoughts.

I get your point. They’re just weird, you know? Did I force you to come to my table? We’re good as is. Sure. You’re right.

Hey, so do you wanna go to the dance together or are we gonna keep it out?

**Comment by El-Qassam** [comment]
Gotcha. They are very weird. I honestly would not be surprised if they made some horribly devious comment (which I must note if they did I would impale them) because the last time they saw someone of the female gender, it was when their mom tucked them in to a episode of Newsmax. This kid Sam, a girl literally came to ask him how his day was and he was about to forget his booty giddying. And if they somehow don’t do all that, surely they’ll settle for racism — as you saw a demo of Ethan today, even as we were in the Holocaust unit. But my question is, why did you not believe me when I said they were the weirdest humans alive, and yet you don’t wanna see for yourself?

Anyways. Stop replying to multiple questions in the same sentence, its confusing. I don’t know about the dance. The hooligans need me there. Unfortunately, it’s like coaching a team. They’re sucky, but I love them, but they also don’t give me any time to do anything. Maybe we could hang out at the park again or some other place?

To finish off my full-length report… Ready for camp? I got the suitcase packed. Anyways… Imma go watch some Seth Meyers until I feel stupid. And study for Academic Superbowl.

**Comment by AJSagan** [comment]
> Gotcha. They are very weird. I honestly would not be surprised if they made some horribly devious comment (which I must note if they did I would impale them) because the last time they saw someone of the female gender, it was when their mom tucked them in to a episode of Newsmax. This kid Sam, a girl literally came to ask him how his day was and he was about to forget his booty giddying. And if they somehow don’t do all that, surely they’ll settle for racism — as you saw a demo of Ethan today, even as we were in the Holocaust unit. But my question is, why did you not believe me when I said they were the weirdest humans alive, and yet you don’t wanna see for yourself?
> 
> Anyways. Stop replying to multiple questions in the same sentence, its confusing. I don’t know about the dance. The hooligans need me there. Unfortunately, it’s like coaching a team. They’re sucky, but I love them, but they also don’t give me any time to do anything. Maybe we could hang out at the park again or some other place?
> 
> To finish off my full-length report… Ready for camp? I got the suitcase packed. Anyways… Imma go watch some Seth Meyers until I feel stupid. And study for Academic Superbowl.

Lol. I honestly just wanted a completely fair representation of them, but after what I heard today, I’ll be spared. But what I want to ask you, is what do they think of me, or who do they think I am? Characteristics and everything.

Maybe, some other time we can go to the park. For now, we can have fun at camp. Did you switch tickets for the rooms?

**Comment by El-Qassam** [comment]
I did.

**Comment by El-Qassam** [comment]
Can’t wait. For now, I’m stuffing my brain for Academic Supa Bowl.

**Comment by AJSagan** [comment]
Why are you still up, Mr. Kamal? You should be asleep by 9:30, tsk tsk

**Comment by El-Qassam** [comment]
> Why are you still up, Mr. Kamal? You should be asleep by 9:30, tsk tsk

And why are you, Madam Sagan? You should’ve been in seventh heaven by 8:00?

**Comment by El-Qassam** [comment]
I was studying. What are you doing?

**Comment by AJSagan** [comment]
> > Why are you still up, Mr. Kamal? You should be asleep by 9:30, tsk tsk
> 
> And why are you, Madam Sagan? You should’ve been in seventh heaven by 8:00?

No, I think you’re mistaken. Good thing you said it correctly. I couldn’t imagine being called Mrs. Kamal.

**Comment by El-Qassam** [comment]
Cierra la boca, por favor?

**Comment by AJSagan** [comment]
Only if you will too?

**Comment by AJSagan** [comment]
Good night.

**Comment by AJSagan** [comment]
Love you.

**Comment by El-Qassam** [comment]
> Only if you will too?

I will.

**Comment by El-Qassam** [comment]
> Good night. Love you.

Good night. Love you was very slightly zesty, but love you too.

**Comment by AJSagan** [comment]
What in the world does zesty mean?

**Comment by AJSagan** [comment]
Please stop using hoodlum language.

**Comment by El-Qassam** [comment]
> What in the world does zesty mean?

It means like over the top.

**Comment by El-Qassam** [comment]
> Please stop using hoodlum language.

Okay. That was pretty offensive. You stop being a Karen first.

**Comment by El-Qassam** [comment]
Now let me sleep in peace.

**Comment by AJSagan** [comment]
> > Please stop using hoodlum language.
> 
> Okay. That was pretty offensive. You stop being a Karen first.

I’m not a Karen.

**Comment by AJSagan** [comment]
> Now let me sleep in peace.

Fine. Stop spam texting me first.

**Comment by El-Qassam** [comment]
> > > Please stop using hoodlum language.
> > 
> > 
> > Okay. That was pretty offensive. You stop being a Karen first.
> 
> I’m not a Karen.

You act and look like one sometimes.

**Comment by El-Qassam** [comment]
OK.
Good night.

**Comment by El-Qassam** [comment]
Girl, where the heck are you? When I came in they said you were in East hall or whatever, but you’re not there. I’m outside of 513 right now.

**Comment by AJSagan** [comment]
I’m in Dining Hall right now. In a blue shirt. Just down the stairs, straight, and to the left.

**Comment by El-Qassam** [comment]
I c you. I can see you from up here.

**Comment by El-Qassam** [comment]
Hello top of Ashlyn’s head.

**Comment by AJSagan** [comment]
Well hello there.

**Comment by AJSagan** [comment]
Hey Malek, me and my friends just finished dinner, I think we have until 10. You could come down to the room now.

**Comment by El-Qassam** [comment]
Well that wasn’t horrible.

**Comment by El-Qassam** [comment]
Last parts were a bit weird but not too bad.

**Comment by El-Qassam** [comment]
I expected cringe to be honest.

**Comment by AJSagan** [comment]
Yeah. I expected it to be over the top with all the activities, but it wasn’t unbearable.

**Comment by El-Qassam** [comment]
Agreed.

**Comment by AJSagan** [comment]
Hey Malek, I'm delivering a paper to tech right now. Can you go to Mumma's?

**Comment by El-Qassam** [comment]
Idk, maybe? I'll be there if I can. Photography module is horribly annoying.

**Comment by AJSagan** [comment]
Dude. You might as well have just not sent the project.

**Comment by El-Qassam** [comment]
> Dude. You might as well have just not sent the project.

Cierra la boca.

**Comment by AJSagan** [comment]
No
